### PR TITLE
Add manga-ocr

### PR DESCRIPTION
**Pull request by tebasakimod777**
#1160

### CODE REVIEW

**Comment by kyakuno** [comment]
@tebasakimod777 PR、ありがとうございます。ONNXファイルを共有いただければ幸いです。

**Comment by tebasakimod777** [comment]
前にアップロードしました

**Comment by tebasakimod777** [comment]
@kyakuno 
manga-ocr の実行スクリプトの実装についてですが、

- decoderに入力するための`input_ids`の作成
- decoderの出力値(`logits`)から`tokenizer.decode()`に入力するために出力値を変更

で詰まっています。

参考になる情報やコードがあれば、教えてください。

**Comment by kyakuno** [comment]
mangaocrの下記に推論コードがあります。
https://github.com/kha-white/manga-ocr/blob/master/manga_ocr/ocr.py

下記の手順で推論が行われています。

```
        img = img.convert('L').convert('RGB')
        x = self._preprocess(img)
        x = self.model.generate(x[None].to(self.model.device), max_length=300)[0].cpu()
        x = self.tokenizer.decode(x, skip_special_tokens=True)
        x = post_process(x)
```

_preprocessではfeature_extractorが呼ばれています。
feature_extractorは下記で定義されています。

```
self.feature_extractor = AutoFeatureExtractor.from_pretrained('kha-white/manga-ocr-base')
```

これより、feature_extractorは下記にあることがわかります。
https://huggingface.co/kha-white/manga-ocr-base/tree/main

下記を見ると、ViTFeatureExtractorが使用されていることがわかります。
https://huggingface.co/kha-white/manga-ocr-base/blob/main/preprocessor_config.json

画像を224x224にリサイズして、-0.5して/0.5で正規化した後、VitFeatureExtractorを適用しているようです。

VitFeatureExtractorの概要です。
https://qiita.com/m__k/items/541c37242aa29c6309ee

これをどうにかしてONNXにする必要があります。

**Comment by kyakuno** [comment]
近い内容が下記で議論されていました。
https://github.com/huggingface/transformers/issues/14195

**Comment by kyakuno** [comment]
ちょっと難易度が高い気はします。

**Comment by kyakuno** [comment]
マージしない。
